Fixed the issue with the variant of sort button.

ashfaqnisar opened the pull request:
Hey @zxhmike, Awesome work buddy.
In mui5, by default, the button is taking the **primary colour** rather than **no colour** like mui4. 
Due to that, the sort button in the table is using the primary variant rather than the default variant. So fixed the issue with the variant of the button. 
Present: 
![image](https://user-images.githubusercontent.com/20638539/137074527-780f5cc6-531d-4c54-b15a-e9bba784dc4f.png)
Expected: 
![image](https://user-images.githubusercontent.com/20638539/137074584-ae24cbcf-8640-46b2-9920-53287cebceb5.png)

ashfaqnisar committed:
fixed the issue with the sort button.

ashfaqnisar committed:
Fixed the issue with misalignment of the sorting label

ashfaqnisar commented:
The sorted label of the table was not correctly aligned with the content of the table. 
Extra padding(16px) was added in the mui5 version decreased it to 8px. 
Present: 
![Screenshot 2021-10-13 122202](https://user-images.githubusercontent.com/20638539/137082685-e9e8ee87-0728-44b3-bdab-5c817a23a676.png)
Expected:
![image](https://user-images.githubusercontent.com/20638539/137082539-3c40c7ba-f65f-4c9a-9f56-d1d18845e7f2.png)

zxhmike commented:
@ashfaqnisar Thank you so much for the contribution. I checked and it is looking great. Before we merge this PR, would you mind checking if there are other buttons that have the same situation? Thank you.

ashfaqnisar commented:
Hey @zxhmike, would be happy to do that!

ashfaqnisar committed:
Fixed the issue with variants of the textfield,select & FormControl

ashfaqnisar commented:
Hello @zxhmike, 
In the mui5, the **default variant** of the textfield, select & formcontrol is **outlined**. So, changed the variant to **standard**. 

This fixed the [issue](https://github.com/gregnb/mui-datatables/pull/1772#issuecomment-940831570) of the misalignment of the filter component label with the select field.

ashfaqnisar committed:
Centered the search icon with the textfield & other icons

ashfaqnisar commented:
Hey @zxhmike, 
The search icon of the TableSearch was a bit misaligned with the text field & other icons. So, centred it.
This may be misaligned due to the **reduction of the padding** of IconButton from **48px to 40px** in **mui5**. 
I have added images for reference. 
Previous Version: 
![image](https://user-images.githubusercontent.com/20638539/137258258-ba9c7811-c7f0-46b1-905b-06fcfea16662.png)

Updated Version: 
![Screenshot 2021-10-14 104652](https://user-images.githubusercontent.com/20638539/137257947-2a68e2ee-cf98-46cb-8a9d-535c684e6971.png)

ashfaqnisar commented:
Hey @zxhmike, 
Apart from these issues, I was not able to find any other issues for the time being. I think this PR is good to go. Will push a commit, if I find something!

zxhmike commented:
Thank you @ashfaqnisar . The PR is merged.
